irs-api/src/main/java/org/eclipse/tractusx/irs/aaswrapper/job/delegate/RelationshipDelegate.java

```diff
@@ -114,7 +115,7 @@ private void processEndpoint(final Endpoint endpoint, final RelationshipAspect r
             aasTransferProcess.addIdsToProcess(idsToProcess);
             itemContainerBuilder.relationships(relationships);
             itemContainerBuilder.bpns(getBpnsFrom(relationships));
-        } catch (final UsagePolicyPermissionException e) {
+        } catch (final UsagePolicyPermissionException | UsagePolicyExpiredException e) {
```

Comment:
```suggestion
        } catch (final PolicyException e) {
```

now both have the interface so you can just catch on the interface